corehq/apps/api/resources/meta.py

```diff
@@ -1,27 +1,55 @@
 from django.conf import settings
 
 from tastypie.authorization import ReadOnlyAuthorization
-from tastypie.throttle import CacheDBThrottle
+from tastypie.throttle import BaseThrottle
 
 from corehq.apps.api.resources.auth import LoginAndDomainAuthentication
 from corehq.apps.api.serializers import CustomXMLSerializer
+from corehq.project_limits.rate_limiter import (
+    PerUserRateDefinition,
+    RateDefinition,
+    RateLimiter,
+    get_dynamic_rate_definition,
+)
+from corehq.project_limits.shortcuts import get_standard_ratio_rate_definition
 from corehq.toggles import API_THROTTLE_WHITELIST
 
 
+
+api_rate_limiter = RateLimiter(
+    feature_key='api',
+    get_rate_limits=lambda domain: _get_per_user_api_rate_definition(domain)
+)
+
+
+def _get_per_user_api_rate_definition(domain):
+    return PerUserRateDefinition(
```

Comment:
These numbers were not picked with much intention and so if anyone has strong thoughts on a good limit I am open to them. I also didn't use the db backed rate definition, solely using the class in memory because it made testing easier. The db ones are nice because they are easier to change on the fly, but I am not actually sure how much that is needed, and I find their tendency to diverge from the code (as they have in all our other rate limits) confusing, because it makes the code misleading to read.